### PR TITLE
Added the watteco flash'O

### PR DESCRIPTION
**Pull request by J2thatsme**

### CODE REVIEW

File: types/watecco/flash'O/meta.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Flash'O",
```
Comment (mariusgiger) [inline types/watecco/flash'O/meta.json:2]: please make sure that the path does not contain any special characters

File: types/watecco/flash'O/uplink.js [hunk 2]
```diff
@@ -0,0 +1,1022 @@
...
+          decoded.zclheader.max.unity = "seconds";
+        }
+      }
+    } else {
+      var decoded = {};
+
+      /*
+        TODO: No Hardcode
+        This function needs to be filled with all the different channels and indexes depending on the batch report.
+        Maybe make them customfields.
```
Comment (mariusgiger) [inline types/watecco/flash'O/uplink.js:907]: this should be documented somewhere - can it also work without custom fields?

File: types/watecco/flash'O/phase1.schema.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "processingType": "uplink_decoder",
+  "topic": "phase1",
+  "title": "Phase 1",
+  "properties": {
+    "pulse": {
+      "title": "Pulse",
+      "type": "integer",
+      "description": "Raw pulse of the phase"
```
Comment (mariusgiger) [inline types/watecco/flash'O/phase1.schema.json:12]: Pulse increment of the phase?

File: types/watecco/flash'O/uplink.js [hunk 4]
```diff
@@ -0,0 +1,1022 @@
...
+        out.series[sampleIndex].uncompressSamples.push(currentMeasure);
+      }
+    }
+  }
+  return last_timestamp;
+}
+
+function adaptToExpectedFormat(out, argList, batchAbsoluteTimestamp) {
+  const returnedGlobalObject = {
+    // batch_counter: out.batch_counter,
```
Comment (mariusgiger) [inline types/watecco/flash'O/uplink.js:611]: there is some code commented out (also below), is there a reason for that?

**Review by mariusgiger — COMMENTED** [review]
thanks for the PR, a few suggestions

**Review by sevyakenza — APPROVED** [review]

**Pull request merged**